site/app/controllers/grading/SimpleGraderController.php

```diff
@@ -172,6 +172,8 @@ public function gradePage($gradeable_id, $view = null, $sort = null) {
             $graders[$section->getName()] = $section->getGraders();
         }
 
+        $grader = $this->core->getUser();
```

Comment:
Can you not call this in SimpleGraderView.php instead of calling this here and passing the variable through?

Reply:
I'm sorry can you elaborate more?